manifests/registry.pp

```diff
@@ -73,7 +73,7 @@
     $local_user_strip = regsubst($local_user, '-', '', 'G')
 
     $_pass_hash = $pass_hash ? {
-      Undef   => pw_hash("${title}${auth_environment}${auth_cmd}${local_user}", 'SHA-512', $local_user_strip),
+      'Undef'   => pw_hash("${title}${auth_environment}${auth_cmd}${local_user}", 'SHA-512', $local_user_strip),
```

Comment:
@sheenaajay have you run this through travis without the qoutes? The PR to remove them was valid and needs to as is.